src/traces/indicator/plot.js

```diff
@@ -769,17 +823,40 @@ function strTranslate(x, y) {
     return 'translate(' + x + ',' + y + ')';
 }
 
-function fitTextInsideBox(el, width, height) {
+function fitTextInsideBox(textBB, width, height) {
     // compute scaling ratio to have text fit within specified width and height
-    var textBB = Drawing.bBox(el.node());
+    // var textBB = Drawing.bBox(el.node());
```

Comment:
:zap:

Reply:
Done in 68c2aefa8ab6af09c598b3739149e2d5e89155d9